code/game/objects/items/anvil.dm

```diff
@@ -0,0 +1,33 @@
+/** Anvil
+	Is treated as an item that can not be picked up, unless you are incredibly strong.
+
+**/
+
+/obj/item/anvil
+	name = "anvil"
+	desc = "For rounding and crafting objects. Combined with a hammer, you can likely craft some pleasant weapons with this"
+	w_class = W_CLASS_GIANT
+	icon = 'icons/obj/blacksmithing.dmi'
+	icon_state = "anvil"
+	impactsound = 'sound/misc/clang.ogg'
+	flags = FPRINT | TWOHANDABLE | MUSTTWOHAND
+	density = 1
+	throwforce = 40
+
+/obj/item/anvil/can_pickup(mob/living/M)
+	if(!..())
+		return FALSE
+	if((M_HULK in M.mutations) && (M_STRONG in M.mutations))
```

Comment:
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/modules/mob/living/helper_procs.dm#L31

Use ``M.get_strength()`` for all checks in this file